src/test/java/kostka/moviecatalog/service/DbMovieServiceIntegrationTest.java

```diff
@@ -72,25 +73,33 @@ public void doNotCreateMovieWithEmptyDtoIntegrationTest() {
 
     @Test
     public void moviePopulatedFromDtoProperlyIntegrationTest() {
-        Movie movie = new Movie(1L, TEST_NAME, TEST_DIRECTOR, TEST_CAMERA, TEST_CAMERA, TEST_DESCRIPTION, 0);
         MovieDto dto = new MovieDto();
-        dto.setName(TEST_NAME);
+        dto.setName(TEST_NAME_2);
         dto.setCamera(TEST_CAMERA);
         dto.setDescription(TEST_DESCRIPTION);
         dto.setDirector(TEST_DIRECTOR);
         dto.setMusic(TEST_MUSIC);
 
         Movie createdMovie = dbMovieService.createMovie(dto);
 
-        Assertions.assertThat(createdMovie).isEqualToComparingOnlyGivenFields(movie);
+        Assertions.assertThat(createdMovie.getName()).isEqualTo(dto.getName());
+        Assertions.assertThat(createdMovie.getDirector()).isEqualTo(dto.getDirector());
+        Assertions.assertThat(createdMovie.getCamera()).isEqualTo(dto.getCamera());
+        Assertions.assertThat(createdMovie.getMusic()).isEqualTo(dto.getMusic());
+        Assertions.assertThat(createdMovie.getDescription()).isEqualTo(dto.getDescription());
+        Assertions.assertThat(createdMovie.getRating()).isEqualTo(0);
     }
 
     @Test
     public void getExistingMovieIntegrationTest() {
-        Movie movie = new Movie(1L, TEST_NAME, TEST_DIRECTOR, TEST_CAMERA, TEST_CAMERA, TEST_DESCRIPTION, 0);
-        Movie returnedMovie = dbMovieService.getMovie(movie.getId());
+        Movie returnedMovie = dbMovieService.getMovie(1L);
 
-        Assertions.assertThat(returnedMovie).isEqualToComparingOnlyGivenFields(movie);
+        Assertions.assertThat(returnedMovie.getName()).isEqualTo(TEST_NAME);
```

Comment:
Asi otazka osobni preference:
Ale klidne bych importoval primo assertThat a v metodach to volal bez te predpony Assertions. Zkrati ti to kod.

Reply:
jo, to muzu udelat :)